packages/devtools_app/lib/src/screens/memory/panes/allocation_tracing/allocation_profile_class_table.dart

```diff
@@ -121,7 +127,13 @@ class _AllocationTracingTableState extends State<AllocationTracingTable> {
           child: DevToolsClearableTextField(
             labelText: 'Class Filter',
             hintText: 'Filter by class name',
-            onChanged: widget.controller.updateClassFilter,
+            onChanged: (value) {
+              ga.select(
+                analytics_constants.memory,
+                analytics_constants.MemoryEvent.tracingClassFilter,
```

Comment:
will this be triggered on every update to the text field? We probably don't want to send an analytics event for every character change to the text field. Can we send this in onSubmitted or onEditingComplete instead? You'll have to play with this a bit to make sure we send the event at the right time

Reply:
Added throttling to handle this.